### DISCUSSION TITLE
ShellyPlus1PM non integrabile in HA

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#1995] [HUB: Home Assistant]
Salve a tutti, ho provato ad integrare il nuovo acquisto ShellyPlus1PM ma l'itegrazione mi dice non supportato come in allegato.
Nella guida inDomus leggevo che era perfettamente compatibile, dovo settare qualcosa? grazie
![Catturashely](https://user-images.githubusercontent.com/84615011/195585215-d05cc8dd-b49c-41d0-8783-b44d5e8004e9.JPG)

**Comment by snakuzzo** [comment]
Lo Shelly 1 Plus è perfettamente compatibile.
Hai protetto lo Shelly con password ? In tal caso prova a toglierla e a riprovare.

Per tutto il resto... https://www.home-assistant.io/integrations/shelly/#known-issues-and-limitations

**Comment by calz64** [comment]
Grazie, in effetti è bastato aggiornare firmware e tutto è andato, perfettamente compatibile. un saluto
